hrm_main.py

```diff
@@ -0,0 +1,16 @@
+import hrm_class
+
+def main(filename,
+        user_specified_time1=0,
+        user_specified_time2=30,
+        brady_threshold=50,
+        tachy_threshold=100,
+        brady_time=5,
+        tachy_time=5,
+        inst=False,
+        avg=False,
+        ano=False):
+    hrm_object = hrm_class.hrm_data(filename)
```

Comment:
generally, simply instantiating a class should not be performing the whole analysis. (This is assuming that, in this case, your goal was for the `hrm_data` class to be a container for the raw ECG data. If not, then perhaps consider renaming the class--we can dive deeper into this if you want). Consider instantiating a new HRMData object and then you as the user of your class should use the class API to perform some explicit actions on that data container.
```python
hrm = HRMData(filename)
hrm.calculate_heart_rates()
hrm.write_summary_to_file("test.txt")
...
etc
```
You'll notice this general pattern of using a class throughout all the modules you use in python :)